.travis.yml

```
@@ -18,13 +18,29 @@ jobs:
 
   include:
     # Integration tests with scylla
-    #- name: Integration Test
-    #  os: linux
-    #  dist: xenial
-    #  python: 3.7
-    #  script:
-    #    - ./ci/run_integration_test.sh
-    #  if: type = pull_request
+    - name: Integration Test #1
```

I think yaml consider the number you've added as comment
```suggestion
    - name: "Integration Test #1"
```